cli/js/testing.ts

```diff
@@ -104,7 +104,7 @@ export function test(
   let testDef: TestDefinition;
   const defaults = {
     ignore: false,
-    sanitizeOps: true,
+    sanitizeOps: false,
```

Comment:
@ry What is the rationale here? I think defaulting to `true` is good. Otherwise you need to add `sanitizeOps` to every single test. I think the group of tests that should have this enabled is greater than the ones that don't.